template/page-events.html.jinja

```diff
@@ -378,7 +378,7 @@
     { "name": "AMAs: XRPL Developer AMAs",      
     "description": "A chat with Matt Mankins from Lorem Labs to discuss Kudos for Code and his recent XRPL Accelerator acceptance.",
     "type": "ama",
-    "link": "https://discord.com/invite/xrpl",
+    "link": "https://discord.com/invite/NmCkgEe7?event=1126528012553302067",
```

Comment:
Discord is a bit weird. That link is correct for people who aren't already members of the Discord server. But for people who are, I think this one is the event link:

https://discord.com/events/886050993802985492/1126528012553302067

Reply:
I am not even logged in to my discord nor part of the server when I am logged in and it works in both cases.